ethicalads-theme/templates/ea/homepage.html

```diff
@@ -76,7 +75,7 @@ <h3>Reach developers</h3>
         <h3>Focused and relevant</h3>
```

Comment:
Better to be explicit here? 

```suggestion
        <h3>Developer-focused ads</h3>
```